ui-kit/src/main/kotlin/org/p2p/uikit/components/edittext/v2/NewUiKitEditText.kt

```diff
@@ -0,0 +1,48 @@
+package org.p2p.uikit.components.edittext.v2
+
+import androidx.constraintlayout.widget.ConstraintLayout
+import androidx.core.content.res.use
+import android.content.Context
+import android.os.Parcelable
+import android.util.AttributeSet
+import org.p2p.uikit.R
+import org.p2p.uikit.components.edittext.UiKitEditTextSavedState
+import org.p2p.uikit.databinding.WidgetUiKitEditTextNewBinding
+import org.p2p.uikit.utils.focusAndShowKeyboard
+import org.p2p.uikit.utils.inflateViewBinding
+
+class NewUiKitEditText @JvmOverloads constructor(
```

Comment:
to remove breaking changes - there is a new version for UiKitEditText - scalable and adoptive straight from UiKIt.